CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-vc-verifier ChangeLog
 
+## 15.1.0 - 2023-TBD
```

Comment:
There's also a breaking change here that uses the next version of bedrock-did-io.

```suggestion
## 16.0.0 - 2023-TBD
```

Reply:
Updated [97349ac](https://github.com/digitalbazaar/bedrock-vc-verifier/pull/47/commits/97349ac6bd72eb4ea464aaf1dfd88d57ef526c8c)